src/mmw/apps/bigcz/clients/cuahsi/search.py

```diff
@@ -29,10 +29,29 @@
 DATE_MAX = date(2100, 1, 1)
 DATE_FORMAT = '%m/%d/%Y'
 
+GRIDDED = [
+    'NWS-WGRFC_Hourly_MPE',
+    'NWS_WGRFC_Daily_MPE_Recent_Values',
+    'LMRFC_Data',
+    'GLDAS_NOAH',
+    'NLDAS_FORA',
+    'NLDAS_NOAH',
+    'TRMM_3B42_7',
+]
 
 client = Client(CATALOG_URL, timeout=settings.BIGCZ_CLIENT_TIMEOUT)
 
 
+def get_networkIDs(services, gridded=False):
+    if not gridded:
+        return [str(s.ServiceID) for s in services
+                if s.NetworkName not in GRIDDED]
+
+    # If we don't want to filter, send nothing instead of everything
+    # Sending all networkIDs returns 0 results in GetSeriesCatalogForBox2
```

Comment:
I've read this a bunch of times, and I can't quite get a handle on it. It seems like sending all networkIDs is filtering, which is why 0 results are returned?

Reply:
This comment should probably be reworded, but the behavior _is_ counterintuitive.

Assume that there are 5 networkIDs total: `[a, b, c, d, e]`. If we make a request with a subset, like `[a, b, c]` we get results with just those networkIDs. If we make a request with an empty list, `[]`, we get results with all networkIDs. If we make a request with the full list `[a, b, c, d, e]`, we get 0 results.

I will reword this to read easier.